src/main/java/ru/testtask/controller/ChatController.java

```diff
@@ -0,0 +1,52 @@
+package ru.testtask.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.messaging.simp.SimpMessageHeaderAccessor;
+import org.springframework.messaging.simp.SimpMessageSendingOperations;
+import org.springframework.stereotype.Controller;
+import ru.testtask.model.Message;
+import ru.testtask.service.MessageService;
+
+import java.util.List;
+
+import static java.lang.String.format;
+
+@Controller
+public class ChatController {
+    @Autowired
+    private SimpMessageSendingOperations messagingTemplate;
+
+    @Autowired
+    private MessageService messageService;
+
+    @MessageMapping("/chat/{roomId}/sendMessage")
+    public void sendMessage(@DestinationVariable String roomId, @Payload Message message) {
+        if (message.getContent().length() > 1000){
```

Comment:
It is better to decline the message. Client should know that the message cannot be sent. It is better to have possibility to change max message length using application properties. And property should have default value 1000